src/ui/components/DropdownMenuItem/index.js

```diff
@@ -0,0 +1,35 @@
+/* @flow */
+import React from 'react';
+import classNames from 'classnames';
+
+import Link from 'amo/components/Link';
+
+import './styles.scss';
+
+
+type Props = {|
+  children: string | Link,
+  detached?: boolean,
+  onClick?: Function,
+|};
+
+const DropdownMenuItem = ({ children, onClick, detached = false }: Props) => {
+  const childIsComponent = typeof children === 'object';
+  const classnames = classNames('DropdownMenuItem', {
```

Comment:
be careful with overshadowing names like this. I know it will pass lint checks since it has a lower case `n` but this makes it a little confusing to read. I would suggest using a different name like `classes` or `computedClassNames` or `_classNames`.